### PR TITLE
(feat) ZOO-006: create old-men zone

### PR DESCRIPTION
**Pull request by NikaKlokava**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,5 +16,19 @@
       </p>
       <button class="button button-type-1">watch online</button>
     </div>
+
+
+    <div class="old-man">
+      <h2 class="text header-2">The Backstage of the Wilderness World.</h2>
+      <img class="img-men" src="./assets/images/bamboo_cap.png">
+      <span class="text body">The site was founded on the basis of a volunteer movement to protect and care for animals.</span>
```
Comment (DzmitryKlokau) [inline index.html:24]: Use p not spans in this code block

File: index.html [hunk 2]
```diff
@@ -16,5 +16,19 @@
...
+      <span class="text subtitle">How it works</span>
+      <span class="text body-medium">The main goal is to help the animals, as well as the nature reserves and zoos where they are kept. We are currently working on video projects targeting pandas in China, eagles on an island near Los Angeles, alligators in Florida and gorillas in the Congo. These have a total of more than 1,500 mammals and reptiles.</span>
+    </div>
+    <div>
```
Comment (DzmitryKlokau) [inline index.html:28]: Why do you need this div? There are no styles, so probably you can remove this div

File: index.css [hunk 3]
```diff
@@ -33,3 +33,54 @@ body {
   top: 626px;
   margin: 0px;
 }
+
+.old-man {
```
Comment (DzmitryKlokau) [inline index.css:37]: Could you use absolute stile for this class?

File: index.css [hunk 4]
```diff
@@ -33,3 +33,54 @@ body {
...
+    padding-top: 140px;
+    padding-left: 100px;
+}
+.old-man > .img-men {
+    width: 609px;
+    height: 403px;
+}
+
+.old-man > .header-2 {
+    height: 116px;
```
Comment (DzmitryKlokau) [inline index.css:47]: Could you remove all widths and height for all text classes? Move it the dov styles?

**Review by DzmitryKlokau — CHANGES_REQUESTED** [review]

**Review by Dzmitry-Klokau — APPROVED** [review]

**Pull request merged**